modules/system/controllers/updates/_list_toolbar.htm

```diff
@@ -11,6 +11,11 @@
         class="btn btn-success oc-icon-plus">
         <?= e(trans('system::lang.plugins.install')) ?>
     </a>
+    <a
+        href="<?= Backend::url('system/updates/install/themes') ?>"
+        class="btn btn-success oc-icon-plus">
+        <?= e(trans('system::lang.themes.install')) ?>
+    </a>
```

Comment:
Just move this before the install plugins button and it should be good to go

Reply:
Done.